### ISSUE TITLE
Add DEPOT_PATH[2]/environments/v#.# to the LOAD_PATH by default

### ISSUE THREAD

**Issue by mauro3** [JuliaLang/Pkg.jl#2246]
After doing some system wide installs (#2245), I'm wondering why `DEPOT_PATH[2]/environments/v#.#` is not part of the default load path.  My understanding is that `DEPOT_PATH[2]` is the normal place for system-wide installs and is there  by default.  So it seems strange that the related environment is not there by default.

**Comment by KristofferC** [comment]
Just noting that this is technically more of a Julia issue than a Pkg issue. Pkg does not decide what is in `LOAD_PATH`.

**Comment by StefanKarpinski** [comment]
If you don't have a `v#.#` environment in `DEPOT_PATH[1]` then it should be.

**Comment by mauro3** [comment]
Yes, but usually one has a `v#.#` in DEPOT_PATH[1].  Otherwise the user can't install anything themselves.  I argue here that there should be both present, as stacked environment.

**Comment by StefanKarpinski** [comment]
There can only be one environment named `v1.5` so there's no way to refer to "the other `v1.5` environment" if you have one in `DEPOT_PATH[1]` already. You can create `DEPOT_PATH[2]/environments/v#.#-system` and then put whatever you want in there. You can also change the user's default load path by arranging for the variable `JULIA_LOAD_PATH="@:@v#.#:@v#.#-system:@stdlib"` to be set in their environment before they log in. Then users will see the named system environment in their default load path. They can still modify the load path further as long as they do it using the recommended `export JULIA_LOAD_PATH="/some/path:$JULIA_LOAD_PATH"` pattern.

**Comment by fredrikekre** [comment]
> There can only be one environment named v1.5 so there's no way to refer to "the other v1.5 environment" if you have one in DEPOT_PATH[1] already.

Technically `v#.#` could expand to multiple entries, e.g. `DEPOT_PATH[1]/environments/v1.5`, `DEPOT_PATH[2]/environments/v1.5` though.

**Comment by StefanKarpinski** [comment]
It could, but then it's meaning would be inconsistent between `LOAD_PATH` and say `] activate @v1.5`. Although maybe that's ok since activation is only about modification and modification affects only the top entry in a stack? Would we do the same thing with other named environments? Would `@devtools` in the load path actually expand to all environments named `devtools`? I.e. a stack of `$depot/environments/devtools` for all `depot` in `DEPOT_PATH`?

**Comment by StefanKarpinski** [comment]
It would be a bit weird if there was a secondary `@v1.5` environment that you can load things from but that you have no way to activate...

**Comment by fredrikekre** [comment]
Yea, I don't know whats best.

Perhaps to reformulate the problem; there is no default system entry in the load path, so for a global install you have to modify each users `JULIA_LOAD_PATH` if you want a set of system global packages. In contrast; there is a default system entry in the depot path where you can download packages and artifacts and there is no need to modify `JULIA_DEPOT_PATH`. Perhaps they don't have to "agree", but it is slightly annoying to have to modify `JULIA_LOAD_PATH` for all users I suppose.
